src/model/users/login.entity.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { User } from './user.entity';
+import * as bcrypt from 'bcrypt';
+
+@Entity({ name: 'user_login_v1' })
+export class Login {
```

Comment:
Question.
1. Why do you need to have the `Login` table? What is the use case?
2. Do you think if it is necessary to add the `status` field?

Reply:
after discussion, keep the original version